server/app/api.py

```diff
@@ -674,9 +684,6 @@ def get_backups(self, obj, user, data):
     def get_submissions(self, obj, user, data):
         return [subm.submission for subm in obj.get_submissions(data['assignment'], data['quantity'])]
 
-    def timed_submission(self, obj, user, data):
-        return obj.get_submission_before(data['assignment'], data['before'])
```

Comment:
Er, didn't find it in the codea nd it wasn't exposed as an endpoint. Unsure though.